SumoLogic.Logging.Common/Sender/SumoLogicMessageSender.cs

```diff
@@ -261,48 +259,56 @@ public async Task TrySend(string body, string name, string category, string host
                 }
                 catch (AggregateException ex)
                 {
+                    var firstException = ex.Flatten().InnerExceptions?.FirstOrDefault() ?? ex.InnerException ?? ex;

```

Comment:
Same here, can we get `AggregateException` here?

Reply:
When awaiting a task, then all exceptions are wrapped in `AggregateException`. And here we  `await httpClient.PostAsync(...)`

Reply:
Sorry you are right. When not using Task.Wait() but instead using await, then it will NOT throw AggregateException. Will fix the code to match new expectations.